recipes/autorom-accept-rom-license/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "AutoROM-accept-rom-license" %}
```

Comment:
Why is this package needed? The README says that users should accept the license by calling

```bash
AutoROM --accept-license
```
It's not OK to have a package which silently accepts the license for the user. Downstream packages could depend on this package and accept the license for them automatically and silently.

Reply:
I just tried to bring some dependencies of gymnasium to conda-forge. This is the equivalent of PyPI's [autorom[accept-rom-license]](https://pypi.org/project/AutoROM.accept-rom-license/). There it is stated, that you can accept the license either with `AutoROM --accept-license` or by installing `autorom[accept-rom-license]`. If it is used in a dependency, the dependency should also have the extra `[accept-rom-license]`, like with [gymnasium](https://github.com/Farama-Foundation/Gymnasium/blob/main/pyproject.toml#L38). There, `autorom[accept-rom-license]` is [explicitly](https://github.com/Farama-Foundation/Gymnasium/blob/main/pyproject.toml#L56) excluded from `[all]`, as this would indeed be not OK.

However, this will be obsolete quite soon, as the ROMs will be included in ALE-py, as @pseudo-rnd-thoughts stated. So, if you want, you can close it.